examples/CIB/ex1/main.cpp

```diff
@@ -399,6 +495,83 @@ main(int argc, char* argv[])
             pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
             pout << "\n";
 
+            // Fill ghost cells of pressure and velocity to compute hydrodynamic forces.
```

Comment:
I am a little concerned about relying on the ghost cells having been properly filled in advance. Perhaps you could make two versions of the function, one that fills ghost cells and another that doesn't. This is what I do in HierarchyMathOps --- if you don't tell it how to fill ghost cells, it will "trust" that you know what you're doing.

If you only plan to use this class internally in the library, then I think it is OK to require callers to have correctly filled ghost cells. If you want it to show up in user code, then I would prefer if it is able to do something reasonable automatically for ghost cells.

Does this make sense?